cmd/exec/main.go

```diff
@@ -49,12 +48,10 @@ func main() {
 	rootCmd.AddCommand(execCmd)
 	rootCmd.AddCommand(pushCmd)
 	rootCmd.AddCommand(setCmd)
-	rootCmd.AddCommand(eventCmd)
-	rootCmd.AddCommand(eventfilterCmd)
-	eventfilterCmd.AddCommand(createCmd)
-	eventfilterCmd.AddCommand(deleteCmd)
-	eventfilterCmd.AddCommand(updateCmd)
-	eventfilterCmd.AddCommand(applyCmd)
+	rootCmd.AddCommand(eventsCmd)
+	eventsCmd.AddCommand(sendEventCmd)
+	eventsCmd.AddCommand(setFilterCmd)
```

Comment:
I merged the create and update commands into a single command called set-filter. By default it uses the create API, but if you use `-f` or `--force` and it fails because the filter already exists, it will retry immediately using the Update API instead.